Sources/SwiftParser/Statements.swift

```diff
@@ -1240,6 +1280,10 @@ extension Parser.Lookahead {
       .switchKeyword?,
       .poundAssertKeyword?:
       return true
+    case .tryKeyword?:
+      // strip the try and then check if the rest is a statement
+      self.eat(.keyword(.try))
+      return isStartOfStatement(allowRecovery: allowRecovery)
```

Comment:
this seems to resolve all but one of the recovery regressions

Reply:
That’s how I would have implemented it as well. What’s the recovery recession that still remains?

Reply:
it was a bug fixed by the precedence chance where it was doing ")blah return foo" and not recovering to the return

Reply:
Ah, great 👍